NitroxClient/MonoBehaviours/DiscordRP/DiscordRpc.cs

```diff
@@ -0,0 +1,205 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+using System.Text;
+using AOT;
+
+public class DiscordRpc
```

Comment:
This class isn't in a namespace. I think it's best to move all the Discord specific classes (including AcceptResponse class) into a `NitroxClient.Discord` namespace.

Reply:
Yeah was there. Gets lost after update because I replaced it.